examples/graph_example.ml

```diff
@@ -76,1095 +76,40 @@ and t = transp
+      [| n; o; o; o; o; o; o; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; t |];
+      [| n; o; o; o; o; o; o; o; n; n; n; g; g; g; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; t; t |];
+      [| n; n; o; o; o; o; o; o; o; n; n; n; c; c; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; t; t |];
+      [| t; n; n; o; o; o; o; o; o; o; n; n; n; c; n; n; n; n; n; n; n; b; b; n; n; n; n; n; n; t; t; t |];
+      [| t; t; n; n; n; o; o; o; o; o; o; n; n; n; n; n; n; n; n; n; b; b; b; b; n; n; n; n; t; t; t; t |];
+      [| t; t; t; t; n; n; o; o; o; o; o; o; n; n; n; n; n; n; n; n; b; b; b; b; b; b; n; n; t; t; t; t |];
+      [| t; t; t; t; t; n; n; o; o; o; o; o; o; n; n; n; n; n; n; o; o; b; b; b; b; b; b; n; n; t; t; t |];
+      [| t; t; t; t; t; n; n; o; o; o; o; o; o; b; b; b; b; b; n; n; o; o; b; b; b; b; b; b; n; n; t; t |];
+      [| t; t; t; t; n; n; n; o; o; o; o; o; b; b; b; b; b; b; b; n; n; o; o; b; b; b; b; b; b; n; n; t |];
+      [| t; t; t; t; n; n; n; o; o; o; o; b; b; b; b; b; b; b; b; b; n; n; o; o; b; b; b; b; b; b; n; n |];
+      [| t; t; t; t; n; n; n; o; o; o; o; b; b; b; b; b; n; n; b; b; b; n; n; o; o; b; b; b; b; b; n; n |];
+      [| t; t; t; t; n; n; n; o; o; o; o; b; b; b; b; b; n; n; b; b; b; b; n; n; o; o; b; o; b; b; n; n |];
+      [| t; t; t; t; n; n; n; o; o; o; o; b; b; b; b; b; n; n; b; b; b; b; b; n; n; o; o; o; o; o; n; n |];
+      [| t; t; t; t; n; n; n; o; o; o; o; b; b; b; b; b; n; n; b; b; b; b; b; b; n; n; o; o; o; o; n; n |];
+      [| t; t; t; t; n; n; n; o; o; o; o; o; b; b; b; b; n; n; b; b; b; b; b; b; b; n; n; o; o; n; n; n |];
+      [| t; t; t; t; n; n; n; n; o; o; o; o; o; b; b; b; n; n; n; b; b; b; b; b; b; b; n; n; o; n; b; n |];
+      [| t; t; t; t; t; n; n; n; o; o; o; o; o; o; b; b; n; n; n; b; b; b; b; b; b; b; b; n; n; n; b; n |];
+      [| t; t; t; t; t; t; n; n; o; o; o; o; o; o; o; y; v; y; n; b; b; b; b; b; b; b; b; n; n; b; b; n |];
+      [| t; t; t; t; t; t; t; n; o; o; o; o; o; v; y; o; o; n; n; n; b; b; b; b; b; b; b; n; n; b; b; n |];
+      [| t; t; t; t; t; t; t; n; o; o; o; y; v; o; o; o; o; n; n; n; n; b; b; b; b; b; b; n; n; b; b; n |];
+      [| t; t; t; t; t; t; n; n; o; v; y; o; y; o; o; o; o; o; o; n; n; n; b; b; b; b; b; n; n; b; b; n |];
+      [| t; t; t; t; t; t; n; o; y; y; o; o; v; o; o; o; o; o; o; o; n; n; n; b; b; b; n; n; n; b; n; t |];
+      [| t; t; t; t; t; n; n; v; o; v; o; o; o; o; o; o; o; o; o; o; o; n; n; n; b; n; n; n; n; b; n; t |];
+      [| t; t; t; t; t; n; v; o; o; v; o; o; o; o; o; o; o; o; o; o; o; o; n; n; n; n; n; n; n; n; t; t |];
+      [| t; t; t; t; n; n; o; o; o; o; o; o; o; o; o; o; o; o; o; o; o; n; n; n; n; n; n; t; t; t; t; t |];
+      [| t; t; t; t; n; o; o; o; o; o; o; o; o; o; o; o; o; o; o; o; n; n; t; t; t; t; t; t; t; t; t; t |];
+      [| t; t; t; t; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; n; t; t; t; t; t; t; t; t; t; t; t |];
     |]
+    [@@ocamlformat "disable"]
```

Comment:
@gpetiot I think it would be useful to be able to set the margin as an attribute in this case, but is there any reason that it cannot be used?